keyboard-compose/src/androidLibMain/kotlin/extensions.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ Copyright 2021 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+package com.splendo.kaluga.keyboard.compose
+
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+
+val LocalFocusRequester = staticCompositionLocalOf { FocusRequester() }
+
+fun Modifier.gainFocus(): Modifier = composed {
+    focusRequester(LocalFocusRequester.current)
+}
```

Comment:
TBH, I'm not sure there should be anything on top of the compose focus API, because it's already quite extensive. Yes, it saves a user from the necessity to pass one argument (`FocusRequester`), but it also adds-up to the public API. If that is about `FocusRequester` instantiation, then it already exists `FocusRequester.Default`.

Reply:
6a616ca c41ff81 0dacb34